### PR TITLE
Audi Issue 12: Add foreignTokenIDOf to IGateway

### PR DESCRIPTION
**Pull request by alistair-singh**
Resolves: SNO-1204

### CODE REVIEW

File: contracts/src/interfaces/IGateway.sol [hunk 1]
```diff
@@ -86,6 +86,9 @@ interface IGateway {
     /// @dev Check whether a token is registered
     function isTokenRegistered(address token) external view returns (bool);
 
+    /// @dev Get token id an ERC20 contract address.
```
Comment (claravanstaden) [inline contracts/src/interfaces/IGateway.sol:89]:
```suggestion
    /// @dev Get token id of an ERC20 contract address.
```

File: contracts/src/interfaces/IGateway.sol [hunk 2]
```diff
@@ -86,6 +86,9 @@ interface IGateway {
...
+    function tokenForeignIDOf(address token) external view returns (bytes32);
```
Comment (claravanstaden) [inline contracts/src/interfaces/IGateway.sol:90]: nit:

```suggestion
    function foreignTokenIDOf(address token) external view returns (bytes32);
```

**Review by claravanstaden — APPROVED** [review]

File: contracts/src/interfaces/IGateway.sol (hunk 2 above)
Comment (alistair-singh) [reply contracts/src/interfaces/IGateway.sol]: Addressed in [38ea2b9](https://github.com/Snowfork/snowbridge/pull/1314/commits/38ea2b967413909cc827ac3a8f9c3396e2ccc277)

File: contracts/src/interfaces/IGateway.sol (hunk 1 above)
Comment (alistair-singh) [reply contracts/src/interfaces/IGateway.sol]: Addressed in [38ea2b9](https://github.com/Snowfork/snowbridge/pull/1314/commits/38ea2b967413909cc827ac3a8f9c3396e2ccc277)

**Review by vgeddes — APPROVED** [review]

**Pull request merged**